src/components/Attributes/Attributes.tsx

```diff
@@ -21,14 +21,18 @@ import { defineMessages, FormattedMessage, useIntl } from "react-intl";
 import AttributeRow from "./AttributeRow";
 import { AttributeRowHandlers, VariantAttributeScope } from "./types";
 
+interface AttributeValueFragmentWithRichText extends AttributeValueFragment {
```

Comment:
At this point, it makes sense to just create a fragment that extends AttributeValueFragment and import the output type instead.

```
fragment AttributeValueDetailsFragment on AttributeValue {
        ...AttributeValueFragment
        richText
}
```